CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log for Dash for Julia
+All notable changes to this project will be documented in this file. This project adheres to Semantic Versioning.
+
+## [UNRELEASED]
```

Comment:
@waralex We should update the CHANGELOG here to reflect the current version and its release date.